tests/tools/plugins/cert_update.cc

```diff
@@ -0,0 +1,86 @@
+/** @file
```

Comment:
Do we need this plugin to be in both example/cert_update/ and in tests/tools/plugins/ ?

Reply:
I think building example plugins is not by default? The test would have to compile it and that's why I put it in both places.

Reply:
It would be nice if the tests could copy in or build files from example.  I thought that is what we were doing with null_transform.c, but I see that nearly the same file is in tests/tools/plugin and example/null_transform.  Perhaps a separate effort to remove duplicates.

Reply:
I can try to do it in this PR and make a separate PR to fix null_transform?